app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -1299,9 +1299,7 @@ class AdvancedFilter extends React.Component {
           </Row>
         </Modal.Body>
         <Modal.Footer className="justify-unset">
-          <p className="lead mr-auto">
-            Filter will be applied to the line lists in the <u>{this.props.workflow}</u> workflow until reset.
-          </p>
+          <p className="lead mr-auto">Filter will be applied to all line lists in the current Dashboard until reset.</p>
```

Comment:
does dashboard need to be caps here?

Reply:
Probably not

Reply:
https://github.com/SaraAlert/SaraAlert/pull/937/commits/6f8d9f1349047c6ee2f033f003f86916013aaddf#diff-b16a4080191131abd62e13c342100314e27e185f75ec98667d6635351bfb3ea8L1302-R1302